src/react-components/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import { uid } from "react-uid";
+import { LinkTypes, LinkIconPositions } from "./LinkTypes";
+import { element } from "prop-types";
+import Icon from "../Icons/Icon";
+import { iconRotationTypes } from "../Icons/IconTypes";
+
+export interface LinkProps {
+    /** Controls the link visuals—action, button, or default. */
+    linkType?: LinkTypes;
+
+    /** Href attribute */
+    url?: string;
+
+    /** Determines if icon appears on the right or left */
+    iconPosition?: LinkIconPositions;
+    iconRotation?: iconRotationTypes;
+
+    /** An option to pass an entire className over blockName */
+    className?: string;
+
+    /** BEM specifiers */
+    modifiers?: string[];
+
+    /** BEM specifiers */
+    blockName?: string;
+
+    /** Additional attributes, such as rel=nofollow, to pass to the <a> tag */
+    attributes?: {};
+}
+
+export default class Link extends React.Component<LinkProps, {}> {
```

Comment:
since there's no state change I think you can make this into a functional component